### ISSUE TITLE
HUSKY_SKIP_INSTALL=true don't skip install and return a null error

### ISSUE THREAD

**Issue by williangd** [typicode/husky#541]
I'm trying to skip automatic installation of Git hooks. I tested with `HUSKY_SKIP_INSTALL=true` and `HUSKY_SKIP_INSTALL=1`, but it still tries to install and returns a null error.

```
$ echo $HUSKY_SKIP_INSTALL
true
$ npm i

> husky@3.0.0 install /builds/williangd/husky-test/node_modules/husky
> node husky install

husky > Setting up git hooks
null
husky > Failed to install
```

Link to test project with same problem: https://gitlab.com/williangd/husky-test/

**Comment by williangd** [comment]
`node:alpine` image does not have git installed by default. With git installed the error message is displayed correctly:
```
husky > Setting up git hooks
fatal: not a git repository (or any of the parent directories): .git
husky > Failed to install
```

**Comment by typicode** [comment]
Thanks for the update!

I've just published `v3.0.1` it should show a better error message and not simply null.
